features/topology-map/plugins/org.opennms.features.topology.plugins.topo.application/src/main/java/org/opennms/features/topology/plugins/topo/application/ApplicationTopologyProvider.java

```diff
@@ -62,6 +66,12 @@ public ApplicationTopologyProvider(ApplicationDao applicationDao) {
 
     private void load() {
         graph.resetContainer();
+        // TODO: Patrick: discuss with mvr: why would we transfer one graph into another? I am missing something here?
+        // my assumption was that we use the graph coming from the graph service directly...
+
+        // TODO: Patrick: will be generic later
```

Comment:
@mvrueden let's discuss when you are back. I have the feeling I am missing something here...